### DISCUSSION TITLE
How does a NASA developer gain Open-Source Contributor access to Github Pilot?

### DISCUSSION THREAD

**Discussion by darith27** [nasa/nasa.github.io#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to help guide NASA developers to obtain open-source contribution level access to Github Copilot under the Github NASA organization. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Virendragupta224** [comment]
thanks evevybody
